src/metabase/lib/schema.cljc

```diff
@@ -109,14 +109,23 @@
        (= ref-type (first x))
        (not (contains? valid-ids (get x 2)))))
 
+(defn- stage-with-joins-and-namespaced-keys-removed
```

Comment:
I understand that using `Offset` in custom column expression requires presence of an order-by clause or a breakout clause. The following video shows me trying it out:

- 0:15 - adding order by clause does not change anything, query still fails
- 0:25 - editing a custom column adds `effective-type: "type/Float"` in pMBQL and BE throws because of it

Are those BE issues?

https://github.com/metabase/metabase/assets/6830683/be8a4708-f9b1-4d0f-abd7-98af46b63c0d

Reply:
Aforementioned issues seem to be fixed now :heavy_check_mark: 

Sorry, I was using `42318-offset-custom-columns-frontend` branch to test it.